recipes/date/all/conanfile.py

```diff
@@ -23,7 +35,51 @@ def source(self):
         for patch in self.conan_data["patches"][self.version]:
             tools.patch(**patch)
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["ENABLE_DATE_TESTING"] = False
+        if self.settings.os == "Windows":
+            cmake.definitions["USE_TZ_DB_IN_DOT"] = False
+            cmake.definitions["USE_SYSTEM_TZ_DB"] = False
```

Comment:
If `use_tz_db_in_dot` and `use_system_tz_db` options can't be enabled on Windows, they shoud be conditionally removed in configure.

Reply:
yep, applied.